helm/opendistro-es/templates/configurer/configurer.yaml

```diff
@@ -0,0 +1,57 @@
+{{- if .Values.configurer.enabled }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ template "opendistro-es.fullname" . }}-configurer
+  labels:
+    app: configurer
+{{ include "opendistro-es.labels.standard" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": {{ .Values.configurer.helm.hook }}
+    "helm.sh/hook-weight": "-5"
```

Comment:
Can you add a comment so we don't forget that this is "linked" to the configmap?

Reply:
Do you mean like?
```
# Use higher value so that the configmap is create before the job
"helm.sh/hook-weight": "-5"
```

Reply:
Yes, perfect!

Reply:
Done